snuba/snapshots/postgres_snapshot.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+import jsonschema  # type: ignore
+import json
+import logging
+
+from contextlib import contextmanager
+from dataclasses import dataclass
+from os import path as os_path
```

Comment:
I think this would be clearer as:

```suggestion
import os.path
```

Invocations then can replace `os_path` with `os.path` below.